process/README.md

```diff
@@ -0,0 +1,22 @@
+# TAC Process Documentation #
+
+This folder contains the policies and procedures the OpenSSF TAC uses to perform their duties as well as guidelines for the Technical Initiatives of the Foundation.
+
+## TAC Election Process ##
+- [TAC & SCIR Election Process](https://github.com/ossf/tac/blob/main/elections/tac-and-scir-election-process.md) - revision forthcoming
+- [TAC Elections Folder](https://github.com/ossf/tac/tree/main/elections) - contains documentation and historic election results.
+
+
+## TAC Decision Process ##
+[TAC Decision Process](https://github.com/ossf/tac/blob/main/process/TAC-Decision-Process.md)
+
+## Technical Initiative Processes & Lifecycle ##
+- [Technical Initiative Lifecycles](https://github.com/ossf/tac/blob/main/process/Technical_Initiative_Lifecycle.md)
+- [Technical Initiative Temaplates](https://github.com/ossf/tac/tree/main/process/templates)
+- [Technical Initiative "Gives & Gets"](https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md)
+- [Technical Initiative Funding Process](https://github.com/ossf/tac/blob/main/process/TI%20Funding%20Request%20Process.md)
+- [Building an Open Source Community guidelines](https://github.com/ossf/tac/blob/main/process/building-an-open-source-community.md)
```

Comment:
Suggesting a minor re-arrangement of the listed docs to make this a slightly more TI priority oriented list (imo).
```suggestion
- [Technical Initiative Lifecycles](https://github.com/ossf/tac/blob/main/process/Technical_Initiative_Lifecycle.md)
- [Technical Initiative "Gives & Gets"](https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md)
- [Technical Initiative Funding Process](https://github.com/ossf/tac/blob/main/process/TI%20Funding%20Request%20Process.md)
- [Building an Open Source Community guidelines](https://github.com/ossf/tac/blob/main/process/building-an-open-source-community.md)
- [Technical Initiative Lifecycle Templates](https://github.com/ossf/tac/tree/main/process/templates)
```